src/app/components/client/toolbar/UserMenu.test.tsx

```diff
@@ -38,20 +38,6 @@ it("opens and closes the user menu", async () => {
   expect(screen.queryByText("example@example.com")).not.toBeInTheDocument();
 });
 
-it("opens and closes the user menu", async () => {
```

Comment:
This test had the exact same name as the one preceding it, and as far as I can see, _is_ exactly the same.

Reply:
Oops — testing it twice does not make things better?

Reply:
100% more testing is 100% more better!